### PR TITLE
refactor leadership model with the new leadership model style

### PR DESCRIPTION
**Pull request by TimOsahenru**
## Issue Link 🔗:

**Issue**: #596 

## Type of Change

- [ ] Bug fix 🐞
- [x] New feature/page
- [ ] Documentation update
- [ ] Other

## Description 📋
Overhaul the leadership section with the new regional leadership model

## Checklist ✅

- [x] Followed the [Code of Conduct](https://github.com/BlackPythonDevs/blackpythondevs.github.io?tab=coc-ov-file) and [Contribution Guide](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/469afd71943738b7209bfe56d5a4e35394fbaafc/CONTRIBUTING.md)
- [x] Ran `pre-commit run --all`
- [x] All tests pass locally
- [ ] Added tests (if applicable)
- [ ] Documentation updated (if applicable)

## Additional Notes & Screenshots

Add any additional notes or comments that might be helpful for the reviewers.

### CODE REVIEW

File: _data/regional_leadership.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- name: Abigail "Afi" Gbadago
+  title: Executor(Ghana)
```
Comment (kjaymiller) [inline _data/regional_leadership.yml]:
```suggestion
  title: Executor(Africa)
```

File: _data/regional_leadership.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  image: https://github.com/AfiMaameDufie.png
+  alt: photo of Afi
+
+- name: Lazuich Ford
+  title: Executor(US/Canada)
```
Comment (kjaymiller) [inline _data/regional_leadership.yml]:
```suggestion
  title: Executor(North America)
```

File: _data/regional_leadership.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  image: /assets/images/bpd_stacked.png
```
Comment (kjaymiller) [inline _data/regional_leadership.yml]:
```suggestion
  image: /assets/images/laz.webp
```

File: _data/regional_leadership.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+- name: Abigail "Afi" Gbadago
+  title: Executor(Ghana)
+  image: https://github.com/AfiMaameDufie.png
```
Comment (kjaymiller) [inline _data/regional_leadership.yml]:
```suggestion
  image: /assets/images/afi.webp
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Updates to the actual roles... Please see suggestions. 

I also have #616 that I'm going to push through that will allow you to use the updated  files

**Comment by TimOsahenru** [comment]
@kjaymiller pushed the suggested changes. Kindly review. Thanks

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Looks good but I just notice you completely removed the global leadership team. 

I and Doreen are still working as the global executor and treasurer.

**Comment by TimOsahenru** [comment]
Oops! I've made another push addressing that issue. Please help review again. Thanks

File: _data/global_leadership.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+- name: Jay Miller
+  title: Executor
+  image: https://github.com/kjaymiller.png
+  alt: photo of Jay
+
+- name: Velda Kiara
+  title: Vice Executor
+  image: https://github.com/VeldaKiara.png
+  alt: photo of Velda
```
Comment (kjaymiller) [inline _data/global_leadership.yml]:
```suggestion
```

Velda is actually stepping down and these roles are effectively taking the place of the Vice Executor Role.

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Last minor change:

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**